src/kayenta/edit/inlineTemplateEditor.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { isEmpty } from 'lodash';
+
+import { HelpField } from '@spinnaker/core';
+
+import FormRow from 'kayenta/layout/formRow';
+import { DISABLE_EDIT_CONFIG, DisableableTextarea } from 'kayenta/layout/disableable';
+import { ICanaryState } from 'kayenta/reducers';
+import * as Creators from 'kayenta/actions/creators';
+import {
+  inlineTemplateValueSelector,
+  transformInlineTemplateForSave,
+} from 'kayenta/selectors/filterTemplatesSelectors';
+
+interface IInlineTemplateEditorStateProps {
+  templateValue: string;
+  transformValueForSave: (value: string) => string;
+}
+
+interface IInlineTemplateEditorDispatchProps {
+  editTemplateValue: (value: string) => void;
+}
+
+export type IInlineTemplateEditorProps = IInlineTemplateEditorStateProps & IInlineTemplateEditorDispatchProps;
+
+export function InlineTemplateEditor({
+  editTemplateValue,
+  templateValue,
+  transformValueForSave,
+}: IInlineTemplateEditorProps) {
+  return (
+    <FormRow
+      label={
+        <>
+          Template <HelpField id="canary.config.filterTemplate" />
```

Comment:
I thought you could pass label="Template" and helpFieldId="canary.config.filterTemplate" as props to FormRow and it would do this for you